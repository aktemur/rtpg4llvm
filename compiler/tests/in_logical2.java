/*expr*/
x == y && x != 0 && x <= 7 && x >= 1 && x < y && x > y;
