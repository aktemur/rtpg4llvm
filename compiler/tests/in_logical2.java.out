mkBinOp(LogicalAnd,
         mkBinOp(LogicalAnd,
            mkBinOp(LogicalAnd,
               mkBinOp(LogicalAnd,
                  mkBinOp(LogicalAnd,
                     mkBinOp(EqEq,
                        new VarExpr("x"),
                        new VarExpr("y")),
                     mkBinOp(NotEq,
                        new VarExpr("x"),
                        mkConst(0))),
                  mkBinOp(LessEq,
                     new VarExpr("x"),
                     mkConst(7))),
               mkBinOp(GreaterEq,
                  new VarExpr("x"),
                  mkConst(1))),
            mkBinOp(Less,
               new VarExpr("x"),
               new VarExpr("y"))),
         mkBinOp(Greater,
            new VarExpr("x"),
            new VarExpr("y")))
